profile.py

```diff
@@ -141,6 +156,13 @@ def __array_finalize__(self, obj):
             return
         self.axis = getattr(obj, 'axis', None)
 
+    def fix_profile(self, coordinate = Axis.x):
+        fun = lambda data: data[coordinate-1]
+        tempdata = [(self.x[ind], self.y[ind]) for ind in range(0,len(self.x))]
```

Comment:
Two questions:
 - do you know "zip" method ?
 - do you know what is behind self.x and self.y (see implementation of Curve) ?